python/Ganga/GPIDev/Lib/GangaList/GangaList.py

```diff
@@ -52,8 +52,8 @@ def makeGangaListByRef(_list, preparable=False):
     result = GangaList()
     if len(_list) == 0:
         return result
-    temp_list = [stripProxy(element) for element in _list]
-    result._list = temp_list
+
+    result._list.extend([stripProxy(element) for element in _list])
```

Comment:
Apologies I think I lost context here in this thread with my last comment I think I mixed up the parent issue with other stuff.